src/service/db/scheduler.rs

```diff
@@ -18,12 +18,18 @@ use infra::{
     errors::Result,
     scheduler::{self as infra_scheduler},
 };
+use serde::{Deserialize, Serialize};
 #[cfg(feature = "enterprise")]
 use {
     infra::errors::Error, o2_enterprise::enterprise::common::infra::config::O2_CONFIG,
     o2_enterprise::enterprise::super_cluster,
 };
 
+#[derive(Default, Serialize, Deserialize, Debug)]
+pub struct DerivedTriggerData {
+    pub period_end_time: i64,
+}
+
```

Comment:
maybe we can cast between String and i64 directly to avoid the serialization/deserialization of the struct?